src/Profile/ProfileForm.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import glamorous from 'glamorous';
+
+const ProfileLabel = glamorous.label({
+  fontFamily: 'sans-serif',
+  fontSize: 15,
+});
+
+const ProfileFormInput = glamorous.input({
+  backgroundColor: 'transparent',
+  font: 'sans-serif',
+  textAlign: 'center',
+  fontSize: 20,
+  height: 40,
+  width: 300,
+  border: '0.5px solid black',
+  borderRadius: 5,
+});
+
+const SubmitButton = glamorous.button({
```

Comment:
@binerys what are your thoughts on having multiple styled components in the same file? I think I remember you saying we should only have one component/file but idk what's the best practice here.

Reply:
Oh hey s/o to [JavaScript twitter](https://twitter.com/wesbos/status/964217400915841025), which led me to this [StackOverflow post](https://stackoverflow.com/questions/42987939/styled-components-organization/42996969#42996969) which I really like.

Reply:
Oh, I think I understand now.  So each different styled component usually gets their own file?

Reply:
Yeah I think that'd be best